CHANGELOG.md

```diff
@@ -1,10 +1,30 @@
 # Fluidcontent Change log
 
+upcoming
+------------------
+
+- :exclamation: Support of [TYPO3 6.2 dropped](https://github.com/FluidTYPO3/fluidcontent/commit/f1df9a4b7ee563335c8f1feeb7925e6f820a7f19)
```

Comment:
@NamelessCoder, got it! And will this branch reused https://github.com/FluidTYPO3/fluidcontent/tree/legacy or a new one created?